src/Umbraco.Web.UI.Client/src/packages/documents/documents/item/document-item-ref.element.ts

```diff
@@ -61,49 +78,47 @@ export class UmbDocumentItemRefElement extends UmbLitElement {
 			.observeRouteBuilder((routeBuilder) => {
 				this._editPath = routeBuilder({});
 			});
-	}
 
-	#isDraft(item: UmbDocumentItemModel) {
-		return item.variants[0]?.state === 'Draft';
+		this.#item.observe(this.#item.unique, (unique) => (this._unique = unique ?? ''));
+		this.#item.observe(this.#item.name, (name) => (this._name = name ?? ''));
+		this.#item.observe(this.#item.icon, (icon) => (this._icon = icon ?? ''));
+		this.#item.observe(this.#item.isTrashed, (isTrashed) => (this._isTrashed = isTrashed ?? false));
+		this.#item.observe(this.#item.isDraft, (isDraft) => (this._isDraft = isDraft ?? false));
 	}
 
-	#getHref(item: UmbDocumentItemModel) {
-		return `${this._editPath}/edit/${item.unique}`;
+	#getHref() {
+		return `${this._editPath}/edit/${this._unique}`;
```

Comment:
Dont we have a PATH_PATTERN for this one?

Reply:
Good point. It is updated.